.github/abstract/How to use SwiftCurrent with UIKit.md

```diff
@@ -1 +1 @@
-SwiftCurrent works well with UIKit. To work correctly it does require `FlowRepresentable`s to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
\ No newline at end of file
+SwiftCurrent works well with UIKit. To work correctly, requires `FlowRepresentable`s to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Comment:
Meaning got lost here I think???

I had to read this several times to understand the change. 

Verbose version:

To work correctly you done gotta make your `FlowRepresentable`s `UIViewController`s, if you make them `UIViews` things'll break, so don't do that.

Reply:
```suggestion
SwiftCurrent works well with UIKit. To work correctly, `FlowRepresentable`s need to be `UIViewController`s and not `UIView`s. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Reply:
what about? 
```suggestion
SwiftCurrent works well with UIKit when `FlowRepresentable`s are `UIViewController`s. `UIView`s are not supported at this time. It comes with [support for Storyboards](using-storyboards.html) with `StoryboardLoadable` and [programmatic views](using-programmatic-views.html). 
```

Reply:
+1

Reply:
@Richard-Gist Sounds good to me!